Gemfile.lock

```diff
@@ -85,7 +85,7 @@ GEM
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
       ruby_dep (~> 1.2)
-    loofah (2.6.0)
+    loofah (2.7.0)
```

Comment:
What led you to change this gemfile?

Reply:
Happened when I did bundle install/bundle update after getting some funky errors when starting up this morning.